railib/utilities.py

```diff
@@ -0,0 +1,97 @@
+
+__all__ = [
+    "to_json"
+]
+
+__SYMBOL_PREFIX = ':'
+__ARRAY_MARKER = '[]'
+__ARRAY_SYMBOL = __SYMBOL_PREFIX + __ARRAY_MARKER
+__EMPTY_ARRAY_MARKER = 'Missing'
+
+def to_json(output: list) -> dict: 
+    '''
+    to_json converts Rel relations to dict object.
+    '''
+    
+    def nested_set(root, keys, value):
+        '''
+        set a value in a dict given a list of nested keys. 
+        '''
+        for i in range(len(keys) - 1):
+            key = keys[i]
+            nextValue = {} if type(keys[i+1]) is str else []
```

Comment:
python does not use camelCase for identifiers

Reply:
all variables now are snake_case 👍